### ISSUE TITLE
question about load time

### ISSUE THREAD

**Issue by novid3333** [FlyTechVideos/BluescreenSimulator#91]
is there any way to configure how fast the complete percent increases? i want it to go very slow.

**Comment by FlyTechVideos** [comment]
Yes! You can now have a factor of up to 10 times as slow. Check out the latest release: https://github.com/FlyTechVideos/BluescreenSimulator/releases/tag/3.4.0
